packages/expo-audio/src/ExpoAudio.ts

```diff
@@ -115,7 +116,14 @@ export async function setIsAudioActiveAsync(active: boolean): Promise<void> {
 
 // @docsMissing
 export async function setAudioModeAsync(mode: Partial<AudioMode>): Promise<void> {
-  return await AudioModule.setAudioModeAsync(mode);
+  const audioMode: Partial<AudioMode> =
+    Platform.OS === 'ios'
```

Comment:
Do we need anything here for `web` platform, or does it support the same as iOS?

Reply:
It supports the same as iOS 👍